EZEE-3247 - Upgrade Guide update (#1228)
ezsystems/developer-documentation: docs/upgrading/5_upgrade_database.md

`psql <database_name> < upgrade/db/postgresql/ezplatform-2.5.latest-to-3.0.0.sql`

### Solr configuration

> If you use Solr as a search engine, make sure that Solr configuration is set to commit Solr index changes directly on Repository updates.

Reviewer: ```suggestion
If you use Solr as the search engine, make sure that Solr configuration is set to commit Solr index changes directly during Repository updates.
```

For more information, see [Solr configuration](../guide/search/solr/#further-configuration).

!!! dxp

    ### Site Factory upgrade

>     For production, it is recommended that you create the `ezsite` and `ezsite_public_access` tables and manually importing their schema definitions.

Reviewer: Recommended or necessary? Then either:
```suggestion
    For production, you should create the `ezsite` and `ezsite_public_access` tables and manually import their schema definitions.
```
or
```suggestion
    For production, create the `ezsite` and `ezsite_public_access` tables and manually import their schema definitions.
```
